documentation/specs/src/economics/public-goods-funding.md

```diff
@@ -1,45 +1,193 @@
+### Constructing the council
+All valid PGF councils will be established multisignature account addresses. These must be created by the intdended parties that wish to create a council. The council will therefore have the discretion to decide what threshold will be required for their multisig (i.e the "k" in the "k out of n").
+
+The council will be resonsible to publish this address to voters and express their desired `spending_cap`. 
+
+A council consisting of the same members should also be able to propose multiple spending caps (with the same multisig address). These will be voted on as separate councils and votes counted separately.
+
+
+### Voting on the council
+Once the council has been constructed, it can be voted on by governance particpants through a vote as structured below:
+
+```rust
+struct OnChainVote {
+    id: u64,
+    voter: Address,
+    yay: proposalVote,
+}
+```
+
+In turn the proposal vote will include the a structure like:
+
+```rust
+HashSet<(Address, u64)>
+```
+
+The structure contains all the counsils voted, where each cousil is specific as a pair `Address` (the enstablished address of the multisig account) and `u64` (spending cap)
+
+These votes will then be used in order to vote for various PGF councils. Multiple councils can be voted on through a vector as represented above.
+
+
+### CLI
+
+TODOs:
+
+- PGF Votes
+- Construction of PGF council multisigs 
+
+### Dealing with ties
+In the rare occurance of a tie, the council with the lower spending_cap will win the tiebreak.
+
+In the case of equal tiebreaks, the addresses with lower alphabetical order will be chosen. This is very arbitrary due to the expected low frequency.
+
+### Ensuring enough votes 
+
+In order for a new PGF council to be elected (and hence halting the previous council's power), $\frac{1}{3}$ of validating power must vote on the "New PGF Council Proposal". Once this condition has been met, majority vote will be decided as to which PGF council will be elected.
+
+### Electing the council
+
+Once the elected council has been decided upon, the established address corresponding to the multisig is added to the `PGF` internal address, and the `spending_cap` variable is stored.
+
+### End of Term Summary
+
+At the end of each term, the council is encouraged to submit a "summary"  which describes the funding decisions the councils have made and their reasoning for these decisions. This summary will act as an assessment of the council and will be the information point for governance to decide whether to re-elect the council. This summary should be made as transparant as pos
+
+## Mechanism
+
+The governance proposal PGFCouncil elects the multisignature account address of the PGF council.
+
+The PGF council members will (by definition) be in charge of a "k of n" multisignature account whereby the $\text{10% inflation} * \text{spending_cap}$ is allocated and spent from the `PGF` internal address.
```

Comment:
to be clear, the multisig does not itself hold the PGF internal funds, it is just allowed to spend them